publish/adapter.debug.js

```diff
@@ -1063,3 +1063,22 @@ if (navigator.mozGetUserMedia) {
     AdapterJS.WebRTCPlugin.defineWebRTCInterface,
     AdapterJS.WebRTCPlugin.pluginNeededButNotInstalledCb);
 }
+
+// It's just for webRTCReady function
+AdapterJS.webRTCReady = function (callback) {
+  if (typeof callback !== 'function') {
+    throw new Error('Callback provided is not a function');
+  }
+
+  if (window.webrtcDetectedBrowser !== 'safari' && window.webrtcDetectedBrowser !== 'IE') {
+    setTimeout(function () {
```

Comment:
@serrynaimo make sense. Tested and works fine. Previously, without the 1sec thing there was some errors (my bad I didn't document it down). Looks like it works without it and makes sense not to have the 1sec because it's already implemented natively.